data/opensource/src/androidMain/kotlin/app/tivi/data/opensource/store/AndroidOpenSourceStore.kt

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023, Google LLC, Christopher Banes and the Tivi project contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package app.tivi.data.opensource.store
+
+import android.app.Application
+import app.tivi.data.opensource.OpenSourceItem
+import app.tivi.data.opensource.OpenSourceState
+import app.tivi.data.opensource.SimpleOpenSourceState
+import java.io.IOException
+import java.io.InputStream
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.emptyFlow
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.Json
+import me.tatarka.inject.annotations.Inject
+
+@Inject
+class AndroidOpenSourceStore(private val context: Application) : OpenSourceStore {
+    private var lastAuthState: OpenSourceState = OpenSourceState.Empty
+
+    override fun fetch(): OpenSourceState {
+        return SimpleOpenSourceState(getOpenSourceItemList())
+    }
+
+    @OptIn(ExperimentalSerializationApi::class)
+    fun getOpenSourceItemList(): Flow<List<OpenSourceItem>> {
```

Comment:
There's no need for this to be a Flow. This should be a suspending function with uses `withContext(dispatchers.io) { ... } `